.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Report Bug
```

Comment:
I might change this to `Bug Report` to better mirror `Feature Request`.

Also might drop the "you have found" below - what if their coworker found it?